src/Layers/xrRenderPC_GL/gl_rendertarget_draw_rain.cpp

```diff
@@ -147,12 +147,12 @@ void CRenderTarget::draw_rain(light& RainSetup)
         // clouds xform
         Fmatrix m_clouds_shadow;
         {
-            static float w_shift = 0;
+            //static float w_shift = 0.0f;
             Fmatrix m_xform;
             //Fvector			direction	= RainSetup.direction	;
             Fvector normal;
             normal.setHP(1, 0);
-            //w_shift		+=	0.003f*Device.fTimeDelta;
+            //w_shift += 0.003f*Device.fTimeDelta;
```

Comment:
This now doesn't correspond to the code style of old commented code :D
```suggestion
            //w_shift		+=	0.003f*Device.fTimeDelta;
```